cryptoki-sys/Cargo.toml

```diff
@@ -12,7 +12,7 @@ repository = "https://github.com/parallaxsecond/rust-cryptoki"
 documentation = "https://docs.rs/crate/cryptoki-sys"
 
 [build-dependencies]
-bindgen = { version = "0.63.0", optional = true }
+bindgen = { version = "0.66.0", optional = true }
```

Comment:
Can you set it to `0.66.1`?
https://github.com/parallaxsecond/rust-psa-crypto/blob/main/psa-crypto-sys/Cargo.toml#L15

Reply:
I can, but the way cargo works is that it will pick up the latest anyways. The bindings were generated with 0.66.1

Reply:
Will do the change in a sec

Reply:
Done